### DISCUSSION TITLE
Changing playback speed in editor shouldn't change pitch

### DISCUSSION THREAD

**Discussion by cihe13375** [ppy/osu#15368] [Q&A]
Currently when changing playback speed in the editor, the pitch of music also changes, making some music sound very weird.
Changing play speed in game (like DT or adjusting replay speed) works fine.

Logs are likely unrelated, but I can provide them if required.
version 2021.1028.0 on win10

https://user-images.githubusercontent.com/33959010/139489493-d2cd0515-b08e-4b50-ae14-eb733ee68c42.mp4

**Comment by bdach** [comment]
This is semi-intentional. Changing the pitch/frequency rather than tempo is done because the tempo adjustment has negative effects for timing. From what I know mappers used to time the track at 25% speed on stable and that is apparently also all sorts of wrong.

See also:
https://github.com/ppy/osu/issues/2393#issuecomment-800854465
https://github.com/ppy/osu/pull/12080#issue-835853256
https://github.com/ppy/osu-stable-issues/issues/298#issuecomment-570576862

**Reply by cihe13375** [reply]
Ok I guess it's fine for timing purposes, but imo it's not so good in actual mapping (where you change playback speed to listen to a section more carefully so you know where to emphasis etc.)
I would say it's still fine with speed 75%, but it starts to get uncomfortable at 50% and more severe at 25% (because the music is modified so heavily that it doesn't sound like the original music anymore, and it's hard to map under such context). And I'm (unluckliy) one of the guys who often use 25% speed for this purpose so that's it.
If we have to rely on the pitch change for timing, I would suggest that the pitch changes only in the timing panel, and it doesn't change in the compose panel. I don't know how difficult to implement this though.

**Reply by bdach** [reply]
I guess that would be an option, but people could still think that their beatmap is mistimed if they play the map at 25% tempo for something else and the tempo adjustment messes with the sampling just enough so that the timing sounds off.

I would suggest muting the audio entirely for your use case but that might be... controversial.
